utils/custom/validate.helpers.js

```diff
@@ -0,0 +1,58 @@
+const isValidDate = dateString => {
+  return !isNaN(Date.parse(dateString));
```

Comment:
According to MDN's article on [Date.parse](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/parse)

>It is not recommended to use Date.parse as until ES5, parsing of strings was entirely implementation dependent. There are still many differences in how different hosts parse date strings, therefore date strings should be manually parsed (a library can help if many different formats are to be accommodated).

So we'd probably want to use a library here maybe something like luxon or date-fns that has immutable dates (so not moment.js) 

https://blog.bitsrc.io/9-javascript-date-time-libraries-for-2018-12d82f37872d

Reply:
Also some weird functionality for this it'll flag the following as a valid date `0001-01-101` and last I checked it wasn't although I'm open to the fact that when it comes to dates in the year 0001 that I'm not exactly an expert.

Reply:
interesting... ok I'll swap it out for some other code I had been testing before I found this.